### ISSUE TITLE
Various issues with colorscheme

### ISSUE THREAD

**Issue by tsaber** [vimperator/vimperator-labs#519]
- Bug report

##### Version: no idea how to look that up. the latest version
Vimperator:	3.13.1 (created: 2016/05/04 01:34:00)
Firefox:	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:47.0) Gecko/20100101 Firefox/47.0

##### Description:

I'm getting ReferenceErrors in white on red. Io is not defined, Application is not defined
Furthermore I can't see what I'm writing in the bottom command line. I had highlight custom stuff in my vimperatorrc to make the font white in the bar. Colorschemes dont work either.

##### Expected behavior:

I want to see what I'm writing. Colorschemes dont work either and I failed to find any explanation on the highlight command

**Comment by gkatsev** [comment]
We have a fix for firefox 47 currently in master. I can try and make a release tonight.

**Comment by tsaber** [comment]
that would be much appreciated!

**Comment by tsaber** [comment]
until then. is it safe to use an older firefox version?

**Comment by gkatsev** [comment]
Yeah, the latest version of vimperator works with the current stable version of firefox (46)

**Comment by tsaber** [comment]
thanks!

**Comment by tsaber** [comment]
Hey my firefox automatically updated to 47 and the same bug occurs. I'll try to downgrade

**Comment by timss** [comment]
@tsaber What's the status on this issue? Can it be closed?
